### DISCUSSION TITLE
Security Onion 2.3.10 does not eject the ISO automatically?

### DISCUSSION THREAD

**Discussion by w0lfcat** [Security-Onion-Solutions/securityonion#2419] [Unsupported Versions] [answered]
I've noticed that SO that does not eject the ISO disc automatically before reboot.

![image](https://user-images.githubusercontent.com/71960097/102855705-17191100-4460-11eb-81a0-a71efafb3ed0.png)

Is this common? The reason I'm asking is certain distro will eject ISO automatically before reboot.

If I did not remove the disk from virtual drive, the installation process will keep going on in an infinity loop.

**Comment by jsavary87** [comment]
You are absolutely right! If you install S.O. 2.3 into virtualbox, once you reach the "initial install complete" screen, you need to force unmount the ISO and mount the guest additions disk. Otherwise you will constantly loop back into install. THIS SOLVES SO MANY PROBLEMS!! Amazing find!

**Comment by dougburks** [comment, marked as answer]
I've created Issue #2436 to take a look at this.
